### PR TITLE
chore(t*): backfill `.repo-metadata.json` files

### PR DESCRIPTION
**Pull request by coryan**
Part of the work for #11519

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11597)

### CODE REVIEW

File: google/cloud/trace/v2/.repo-metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "cloudtrace.googleapis.com",
+    "api_shortname": "cloudtrace",
+    "client_documentation": "https://cloud.google.com/cpp/reference/trace/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/559776",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Stackdriver Trace API",
```
Comment (dbolduc) [inline google/cloud/trace/v2/.repo-metadata.json:9]: Cloud Trace API

File: google/cloud/trace/v1/.repo-metadata.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "cloudtrace.googleapis.com",
+    "api_shortname": "cloudtrace",
+    "client_documentation": "https://cloud.google.com/cpp/reference/trace/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/559776",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Stackdriver Trace API",
```
Comment (dbolduc) [inline google/cloud/trace/v1/.repo-metadata.json:9]: Cloud Trace API

File: google/cloud/tpu/v2/.repo-metadata.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "tpu.googleapis.com",
+    "api_shortname": "tpu",
+    "client_documentation": "https://cloud.google.com/cpp/reference/tpu/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/5308873",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Cloud TPU API",
+    "product_documentation": "https://cloud.google.com/tpu/docs",
```
Comment (dbolduc) [inline google/cloud/tpu/v2/.repo-metadata.json:10]: nit: https://cloud.google.com/tpu

File: google/cloud/tpu/v1/.repo-metadata.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "tpu.googleapis.com",
+    "api_shortname": "tpu",
+    "client_documentation": "https://cloud.google.com/cpp/reference/tpu/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/5308873",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Cloud TPU API",
+    "product_documentation": "https://cloud.google.com/tpu/docs",
```
Comment (dbolduc) [inline google/cloud/tpu/v1/.repo-metadata.json:10]: nit: https://cloud.google.com/tpu

File: google/cloud/tasks/v2/.repo-metadata.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "cloudtasks.googleapis.com",
+    "api_shortname": "cloudtasks",
+    "client_documentation": "https://cloud.google.com/cpp/reference/tasks/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/5433985",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Cloud Tasks API",
+    "product_documentation": "https://cloud.google.com/tasks/docs",
```
Comment (dbolduc) [inline google/cloud/tasks/v2/.repo-metadata.json:10]: nit: https://cloud.google.com/tasks

**Review by dbolduc — APPROVED** [review]

File: google/cloud/trace/v2/.repo-metadata.json (hunk 1 above)
Comment (coryan) [reply google/cloud/trace/v2/.repo-metadata.json]: Fixed.  You may want to send a CL to fix the (future) source of truth for these:

https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudtrace/v1/cloudtrace_v1.yaml

File: google/cloud/trace/v1/.repo-metadata.json (hunk 2 above)
Comment (coryan) [reply google/cloud/trace/v1/.repo-metadata.json]: Also fixed.

**Pull request merged**